offchainreporting2plus/types/config_digest.go

```diff
@@ -21,6 +21,8 @@ const (
 	ConfigDigestPrefixSolana     ConfigDigestPrefix = 3
 	ConfigDigestPrefixStarknet   ConfigDigestPrefix = 4
 	ConfigDigestPrefixMercuryV02 ConfigDigestPrefix = 6
+	ConfigDigestPrefixThreshold  ConfigDigestPrefix = 7
```

Comment:
Can you add a comment here as well: "prefixes for running Threshold/S4 plugins as part of another product under  one contract."

Reply:
Another option is to reserve a range for any "secondary" plugin...